lib/jbuilder/collection_renderer.rb

```diff
@@ -40,16 +12,6 @@ def initialize(obj, scope)
         @scope = scope
       end
 
-      # Rails 6.0 support:
-      def each
-        return enum_for(:each) unless block_given?
-
-        __getobj__.each do |object|
-          @scope.call { yield(object) }
-        end
-      end
-
-      # Rails 6.1 support:
```

Comment:
Is this func still needed then?

Reply:
It is. It really should have said "Rails >= 6.1 support".